tests/tests_fabric/strategies/test_fsdp.py

```diff
@@ -302,13 +308,13 @@ def test_fsdp_load_checkpoint_one_fsdp_module_required(tmp_path):
     # missing FSDP model
     with pytest.raises(ValueError, match="Could not find a FSDP model in the provided checkpoint state."):
         strategy.load_checkpoint(path=tmp_path, state={"other": "data"})
-    with pytest.raises(ValueError, match="Could not find a FSDP model in the provided checkpoint state."):
-        strategy.load_checkpoint(path=tmp_path, state={"model": torch.nn.Linear(3, 3)})
```

Comment:
Why is this removed? I expect there to be an error, because there is no FSDP (sub)module in the state.

Reply:
I simply forgot to add this back